tst.c

```diff
@@ -4,6 +4,26 @@
 #define WRDMAX 128
 #define STKMAX (WRDMAX * 2)
 
+/* This macro is used to replace some repeating pattern for rotate and delete
```

Comment:
Fix grammar error again. Should be "for rotating."

Reply:
"rotate" is a verb.

Reply:
I got it! I have changed the word "rotate" to "rotaing", and also the following word "delete" to "deleting". (Does it right? I apologize for my poor English ...)

Some word spelling error are also fixed.